cli/util.go

```diff
@@ -354,6 +355,10 @@ func natsOpts() []nats.Option {
 		connectionName = "NATS CLI Version " + Version
 	}
 
+	if opts.SkipTls {
+		copts = append(copts, nats.Secure(&tls.Config{InsecureSkipVerify: true}))
```

Comment:
Would this not break specifying certificates ca etc? or are those handled by nats.go and added to this tlsc?